src/modules/users/controller.js

```diff
@@ -218,8 +220,8 @@ const getStatus = async (request, h) => {
     data: {
```

Comment:
I can't get GitHub to highlight or put my suggestion in the right place for this one. But I would suggest refactoring this. First, we can combine extracting and parsing the data. Then I would make the return statement simpler by calling the functions first and using their return values.

```javascript
  const results = await getUserCompanyStatus(userResponse.data)

  const companiesData = results[0].data?.companies ?? []
  const verificationsData = results[1].data ?? []
  const documentHeaders = results[2] ?? []
  const licencesMap = results[3]

  const user = mapUserStatus(userResponse.data)
  const companies = mapCompanies(
    companiesData,
    verificationsData,
    documentHeaders,
    licencesMap
  )

  return {
    data: {
      user,
      companies
    },
    error: null
  }
```